package.sh

```diff
@@ -1,8 +1,31 @@
 #!/bin/sh
 
+#Script for create the plugin artifact
+echo "Travis tag: $TRAVIS_TAG"
+
+if [ "$TRAVIS_TAG" = "" ]
+then
+   TRAVIS_TAG='1.0.0'
```

Comment:
Is this a good idea? Defaulting to 1.0.0 if no tag? Why?

Reply:
Yes, is only for local environmets.